plugins/bit_invader/bit_invader.cpp

```diff
@@ -240,7 +248,7 @@ void bitInvader::samplesChanged( int _begin, int _end )
 void bitInvader::normalize()
 {
 	// analyze
-	float max = 0;
+	float max = 0.0001f;
```

Comment:
This is more of a general question: Why don't we use the machine epsilon `std::numeric_limits<float>::eps()` in places where we want to prevent division by zero? In my opinion this would be a cleaner solution to hardcoding a small value like in `Graph.cpp`s `normalize()` that you mention.

Reply:
Ok. Fixed here with	`std::numeric_limits<float>::epsilon();`